File: app/src/main/java/com/forgoogleplay/bodyfatcalculatorpro/MainActivity.java
package com.forgoogleplay.bodyfatcalculatorpro;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void goToBasicActivity(View view) {
        Intent intent = new Intent(this, BasicActivity.class);
        startActivity(intent);
    }

    public void goToProActivity(View view) {
        Intent intent = new Intent(this, ProActivity.class);
        startActivity(intent);
    }

}

File: app/src/main/java/com/forgoogleplay/bodyfatcalculatorpro/MainActivity.kt
package com.forgoogleplay.bodyfatcalculatorpro

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun goToBasicActivity(view: View?) {
        val intent = Intent(this, BasicActivity::class.java)
        startActivity(intent)
    }

    fun goToProActivity(view: View?) {
        val intent = Intent(this, ProActivity::class.java)
        startActivity(intent)
    }
}